Cargo.toml

```diff
@@ -136,7 +137,7 @@ uuid = { version = "1.3.0", features = ["v4"] }
 zstd = "=0.11.2"
 
 # crypto
-rsa = { version = "0.7.0", default-features = false, features = ["std", "pem"] }
+rsa = { version = "0.7.0", default-features = false, features = ["std", "pem", "hazmat"] }
```

Comment:
Can you add a comment here why hazmat feature is used? "hazmat needed for PrehashSigner in ext/node"